equinox/nn/_embedding.py

```diff
@@ -194,12 +213,14 @@ def precompute_freqs_cis(
     def __call__(
         self,
         x: Float[Array, "seq_length embedding_size"],
+        offset: Int[Array, ""],
```

Comment:
Let's make this `offset: Int[ArrayLike, ""] = 0` so that we have backward compatibility?